ob-pattern-1/README.md

```diff
@@ -0,0 +1,95 @@
+# Helm Chart for deployment of WSO2 Open Banking API Manager with WSO2 Open Banking Key Manager
+
+![WSO2 Open Banking Deployment Pattern 1](images/pattern1.png)
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Quick Start Guide](#quick-start-guide)
+
+## Prerequisites
+
+* In order to use WSO2 Open Banking Helm resources, you need an active WSO2 Open Banking subscription. If you do not possess an active WSO2 Open Banking subscription already, you can contact us from [here](https://wso2.com/solutions/financial/open-banking/).<br><br>
+ 
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
+(and Tiller) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (compatible with v1.10) in order to run the 
+steps provided in the following quick start guide.<br><br>
+
+* An already setup [Kubernetes cluster](https://kubernetes.io/docs/setup/pick-right-solution/).<br><br>
```

Comment:
Broken url, this needs to changed to https://kubernetes.io/docs/setup

Reply:
Fixed in https://github.com/wso2/kubernetes-open-banking/pull/22/commits/801acbc3a7226e2aa43f6fbdeffb5f723f740695